app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -690,10 +691,10 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
                 }
             )
         }
-        CustomizeHome.jumpBackIn.set(settings.showRecentTabsFeature)
-        CustomizeHome.recentlySaved.set(settings.showRecentBookmarksFeature)
+        CustomizeHome.jumpBackIn.set(settings.showRecentTabsPreference)
+        CustomizeHome.recentlySaved.set(settings.showRecentBookmarksPreference)
         CustomizeHome.mostVisitedSites.set(settings.showTopFrecentSites)
-        CustomizeHome.recentlyVisited.set(settings.historyMetadataUIFeature)
+        CustomizeHome.recentlyVisited.set(settings.isHistoryMetadataEnabled)
         CustomizeHome.pocket.set(settings.pocketRecommendations)
```

Comment:
I confused why this section now uses a `show…Preference` rather than a `show…Feature`.

Reply:
Yeah, it's not completely straightforward, I'll add some comments to clarify it. As this code runs on startups for telemetry purpose if we execute `show…Preference` versions, we will be calling the nimbus variables on the first run causing some visual delays, when we just want to sent the user selections (the shared pref values) to the telemetry server.

For example, in the first run we will hit the default value that will hit the nimbus variables. 

```
    var showRecentTabsFeature by lazyFeatureFlagPreference(
        appContext.getPreferenceKey(R.string.pref_key_recent_tabs),
        featureFlag = FeatureFlags.showRecentTabsFeature,
        default = { appContext.components.analytics.features.homeScreen.isRecentlyTabsActive() }
    )
```